docs/architecture-decisions/adr000-use-luxon-locale-and-date-presets.md

```diff
@@ -0,0 +1,41 @@
+---
+id: adrs-adr000
+title: ADR000: Use Luxon.toLocaleString and date/time presets
```

Comment:
Should be the next number in line instead of `000`? Also in the filename. :)

Reply:
Ah it said... 

> Copy `docs/architecture-decisions/adr000-template.md` to `docs/architecture-decisions/adr000-my-decision.md` (my-decision should be descriptive. Do not assign an ADR number.)

Reply:
Was that supposed to be "Do not forget to assign an ADR number" 😅 

I think that's a mistake, but I could be wrong. 
![image](https://user-images.githubusercontent.com/8065913/135838671-03bb1398-8c20-4925-a96d-f6215ec08bb9.png)

Reply:
@jhaals please can you let me know if the ADR number needs adding in this PR or is there some other process after? Getting confused by the  "Do not assign an ADR number." in the guidance? :-)